apps/daimo-mobile/src/view/shared/composeSend.tsx

```diff
@@ -34,20 +30,20 @@ export async function composeEmail(
     const subject = `${senderName} sent you $${dollarStr}`;
     const body = `Hi ${recipientStr},\r\n\r\n${senderName} sent you $${dollarStr} USDC and invited you to join Daimo.\r\n\r\nVisit here to accept: ${url}\r\n\r\nDaimo is a global payments app that lets you send and receive USDC on Ethereum.`;
 
-    const openString = (function () {
-      if (Platform.OS === "android")
-        return `mailto:${email}?subject=${encodeURIComponent(
-          subject
-        )}&body=${encodeURIComponent(body)}`;
-      else return encodeURI(`mailto:${email}?subject=${subject}&body=${body}`);
-    })();
+    // Open email client directly if there's only one
+    const appId = emailClients.length === 1 ? emailClients[0].id : undefined;
 
-    if (await Linking.canOpenURL(openString)) {
-      await Linking.openURL(openString);
-      return true;
-    } else {
-      return false;
-    }
+    const appUsed = await openComposer({
+      defaultEmailLabel: "Apple Mail", // Only used on iOS
```

Comment:
you're sure this is always apple mail?

Reply:
not sure for the edge case where someone has a different default email app (not equal to Apple Mail):

In that case, we'll display the ActionSheet anyway, and worst case clicking this button does the same thing as another button.